django/utils/text.py

```diff
@@ -412,6 +413,17 @@ def camel_case_to_spaces(value):
     return re_camel_case.sub(r' \1', value).strip().lower()
 
 
+def to_path(value):
+    """
+    Convert value to a pathlib.Path instance, if not already a Path.
+    """
+    if isinstance(value, Path):
+        return value
+    elif not isinstance(value, str):
+        raise TypeError('Invalid path type: %s' % type(value).__name__)
+    return Path(value)
```

Comment:
I think that `django.utils._os` would be a better place for this hook. What do you think?

Reply:
It's hard to tell why `_os` :man_shrugging: , we added `_os` twelve years ago (304381616f89745c2b52ba793e5b23184c4788bb). Maybe to avoid confusion with builtin `os` :thinking: .

Reply:
I think so. There was also a `django.utils._decimal` at that time. Remember in Python 2, unless using `from __future__ import absolute_import` (when that became available), imports were relative. Having `django.utils.os` would have prevented other modules in `django.utils` from importing `os` from the standard library.